src/scss/toc.scss

```diff
@@ -8,7 +8,7 @@ $theme: "" !default;
 .md-toc-content {
   margin-left: 2em;
   /* 修复缺失上级标题时无法递增 */
-  counter-reset: toc-h2 toc-h3 toc-h4;
```

Comment:
To **create a new counter**, it is recommended to use the [counter-reset](https://developer.mozilla.org/en-US/docs/Web/CSS/counter-reset) CSS property
See [MDN counter-set](https://developer.mozilla.org/en-US/docs/Web/CSS/counter-set)